class.jetpack-admin.php

```diff
@@ -51,6 +51,14 @@ private function __construct() {
 
 		// Add module bulk actions handler
 		add_action( 'jetpack_unrecognized_action', array( $this, 'handle_unrecognized_action' ) );
+
+		add_action( 'admin_init', array( $this, 'activate_classic_editor' ), 9 );
```

Comment:
placed in this file just to try and find somewhere that it will run and complete before decision on which editor to load is made. Also tried placing in modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php & modules/calypsoify/class.jetpack-calypsoify.php with no joy

Reply:
The problem was that the hook was firing too late. Because of the order in which WordPress actions happen we have to reload to make this work.